plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -151,6 +151,13 @@
             <add-to-group group-id="Github.PullRequest.Changes.Toolbar" anchor="after"
                           relative-to-action="ChangesView.GroupBy"/>
         </action>
+
+        <action id="actions.HideNonFunctionalChangesAction"
+                class="org.jetbrains.research.refactorinsight.actions.HideNonFunctionalChangesAction"
+                icon="RefactorInsightIcons.toggle"
+                text="Hide non-functional Refactorings" description="actions.HideNonFunctionalChangesAction">
```

Comment:
`Hide non-functional changes`
Also, it would be nice to add a tooltip explaining what it means when a user hovers over a checkbox "Refactorings and imports" if it's possible